### DISCUSSION TITLE
Error [ERR_REQUIRE_ESM]

### DISCUSSION THREAD

**Discussion by aureliotechnology** [SBoudrias/Inquirer.js#1270] [Q&A]
I'm getting this error when trying to run a nestjs project with the inquirer lib:

./node_modules/@nestjs/cli/node_modules/inquirer/lib/utils/screen-manager.js:4
const wrapAnsi = require('wrap-ansi');
                 ^

Error [ERR_REQUIRE_ESM]: require() of ES Module ./node_modules/wrap-ansi/index.js from ./node_modules/@nestjs/cli/node_modules/inquirer/lib/utils/screen-manager.js not supported.
Instead change the require of index.js in ./node_modules/@nestjs/cli/node_modules/inquirer/lib/utils/screen-manager.js to a dynamic import() which is available in all CommonJS modules.

Can you help me?

**Comment by SBoudrias** [comment]
Which version of inquirer are you using? Did you try latest v8? `npm install inquirer@^8.0.0`
